README.md

```diff
@@ -154,6 +154,91 @@ const productId = '12345678-1234-1234-4123-123456789012'
 const productTOSUrl = automat.getProductTOS(productId)
 ```
 
+### Branding
+
+Etvas Automat gives you full flexibility about the branding used to display various content. Out of the box, Etvas Automat will ensure the appropriate branding (the one configured in your account on [Partners Studio](https://partners.helloetvas.com)) is used for everything.
+
+Still, there are cases when you will want to alter this default behavior and use a different branding. For this case, Etvas Automat exposes all the methods you need to manage the appearance for the pages or fragments you display.
+
+> **Note** Calling the branding methods requires initialization.
+
+Setting up a custom branding is straightforward: at a minimum, you should setup an `accentColor` in your branding object like this:
+
+```
+// Define the branding
+const branding = {
+  accentColor: '#336699',
```

Comment:
parca aveam 2 culori ... s-a pierdut una pe drum ?

Reply:
Aveam impresia ca e folosit doar `accentColor` si `brandColor` e doar pentru sidebar, dar cred ca in Discover este implicat si `brandColor`. Thanks.